low_cost_robot_arm/low_cost_robot_arm.xml

```diff
@@ -0,0 +1,136 @@
+<mujoco model="low_cost_robot_arm">
+  <compiler angle="radian" meshdir="./assets/"/>
+
+  <option cone="elliptic" impratio="10"/>
+
+  <default>
+    <default class="low_cost_robot_arm">
+      <joint armature="0.5" damping="1.5" stiffness="0" frictionloss="0.2"/>
+      <position kp="100" kv="10" inheritrange="1"/>
+      <default class="visual">
+        <geom type="mesh" group="2" contype="0" conaffinity="0"/>
+      </default>
+      <default class="collision">
+        <geom type="mesh" group="3" mass="0" density="0"/>
+        <default class="pad_box1">
+          <geom type="box" friction="0.7" solimp="0.95 0.99 0.001" solref="0.004 1" mass="0" priority="1" size="0.002 0.002 0.0025" rgba="0.2 0.2 0.2 1"/>
+        </default>
+        <default class="pad_box2">
+          <geom type="box" friction="0.6" solimp="0.95 0.99 0.001" solref="0.004 1" mass="0" priority="1" size="0.002 0.002 0.0052" rgba="0.1 0.1 0.1 1"/>
+        </default>
+      </default>
+    </default>
+  </default>
+
+  <asset>
+    <material name="black" rgba="0.1 0.1 0.1 1" specular="0.1" shininess="0.1"/>
```

Comment:
Nit: I've found that specular and shininess aren't really noticeable in the Menagerie scenes so feel free to remove.